app/Controllers/BaseController.php

```diff
@@ -37,6 +37,13 @@ abstract class BaseController extends Controller
      */
     protected $helpers = [];
 
+     /**
+     * To declare the session propety to allow for global
+     * initialization in the initController() method
+     * See the Creation of dynamic property depreciation i php 8.2 
```

Comment:
```suggestion
     * See the Creation of dynamic property depreciation in php 8.2 
```

Reply:
By the way, `depreciation` hasn't been added to the list to change

Reply:
What do you mean?
Ah, not `depreciation` but `deprecation`.